utils/_context/_scenarios/parametric.py

```diff
@@ -188,6 +189,7 @@ def get_warmups(self):
 
         return result
 
+    @retry(delay=1, tries=5)
```

Comment:
I believe I previously saw failures with short retry times (< 5 seconds), I suggest a longer retry period to wait for whatever the issue is to pass.
```suggestion
    @retry(delay=10, tries=3)
```